lib/SmileIdentityCore.php

```diff
@@ -0,0 +1,289 @@
+<?php
+spl_autoload_register(function($class) {
+     require_once($class.'.php');
+});
+
+use \GuzzleHttp\Client;
+use \GuzzleHttp\Exception\GuzzleException;
+use \Psr\Http\Message\ResponseInterface;
+
+const VERSION = '1.1.0';
+const DEFAULT_JOB_STATUS_SLEEP = 2;
+
+class SmileIdentityCore
+{
+    const SID_SERVERS = [
+        'https://3eydmgh10d.execute-api.us-west-2.amazonaws.com/test',
+        'https://la7am6gdm8.execute-api.us-west-2.amazonaws.com/prod'
+    ];
+    public Signature $sig_class;
+    private String $partner_id;
+    private String $default_callback;
+    private String $sid_server;
+
+    /**
+     * WebApi constructor.
+     * @param $partner_id
+     * @param $default_callback
+     * @param $api_key
+     * @param $sid_server
+     * @throws Exception
+     */
+    public function __construct($partner_id, $default_callback, $api_key, $sid_server)
+    {
+        $this->partner_id = $partner_id;
+        $this->default_callback = $default_callback;
+        $this->sig_class = new Signature($api_key, $partner_id);
+        if(strlen($sid_server) == 1) {
+            if(intval($sid_server) < 2) {
+                $this->sid_server = self::SID_SERVERS[intval($sid_server)];
+            } else {
+                throw new \Exception("Invalid server selected");
+            }
+        } else {
+            $this->sid_server = $sid_server;
+        }
+    }
+
+    public function get_version(): string
+    {
+        return VERSION;
+    }
+
+    /**
+     * @return array
+     */
+    public function generate_sec_key(): array
+    {
+        return $this->sig_class->generate_sec_key();
```

Comment:
There should be an optional parameter when you submit a job allowing the use of the new signature